### DISCUSSION TITLE
what will be if I use `sudo rm -rf /`?

### DISCUSSION THREAD

**Discussion by OstaptsovDanil** [ish-app/ish#2408] [Q&A]
I'm afraid to try it, but I assume that the iPhone will not allow iOS to be taken down

**Comment by carbon-starlight** [comment]
It seems like BusyBox's sudo implementation does not have the `--no-preserve-root` safeguard, so you will likely delete everything in the filesystem of your WM, but not iOS device (that's an emulation, remember?). It would be interesting to try this out though right after fresh installation, maybe I'll try that out on my old iPhone (after I'll fix it) where I didn't install and customize iSH yet to easily reinstall the app after.

**Reply by HVRyan** [reply]
It should not affect your phone at all; everything is contained within the emulation (unless you mount into Files).
